ui/src/store/modules/user.js

```diff
@@ -556,6 +555,9 @@ const user = {
       commit('SET_DOMAIN_STORE', domainStore)
     },
     SetCsLatestVersion ({ commit }, rolename) {
+      if (!vueProps.$config.notifyLatestCSVersion) {
+        return
+      }
```

Comment:
Maybe instead of Github, we should query one of the ASF or repo links to find latest release? For example - https://dlcdn.apache.org/cloudstack/releases/ - we may even introduce to put like a file that gives latest version info say from https://download.cloudstack.org/

Reply:
@rohityadavcloud , I am not sure this would be easily cleanly implemented. 

https://api.github.com/repos/apache/cloudstack/releases give a cleanish yaml with results.

In contrast https://dlcdn.apache.org/cloudstack/releases/ gives a directory listing. 
https://download.cloudstack.org/ also gives a directory listing which is really messy.

I would prefer to continue as is but make the retrieval of new version optional, advising operators to set it on test environments and to disable it in production.

What do you think?

Reply:
can we just put a file on cloudstack.apache.org, for example 

https://cloudstack.apache.org/latest_cloudstack_version ?

We do not need to get the new version very soon after the release, 1-7 days delay is good I think.